libvips/foreign/magick.c

```diff
@@ -80,11 +80,12 @@
 static const char *
 magick_sniff(const unsigned char *bytes, size_t length)
 {
-	if (length >= 4 &&
+	if (length >= 6 &&
 		bytes[0] == 0 &&
 		bytes[1] == 0 &&
-		bytes[2] == 1 &&
-		bytes[3] == 0)
+		(bytes[2] == 1 || bytes[2] == 2) &&
+		bytes[3] == 0 &&
+		(bytes[4] != 0 || bytes[5] != 0)
```

Comment:
Sorry, I accidentally removed a `)` in my previous code suggestion.
```suggestion
		(bytes[4] != 0 || bytes[5] != 0))
```

Reply:
Yeah no worries was about to push the fix